var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -274,37 +272,56 @@ def cmake_args(self):
     #
     # export SPACK_USER_CACHE_PATH=/tmp/spack
     # export SPACK_DISABLE_LOCAL_CONFIG=true
-    def test(self):
-        if not self.spec.satisfies("@develop") or not os.path.isdir(self.prefix.bin):
-            tty.info("Skipping: checks not installed in bin for v{0}".format(self.version))
-            return
-
-        tests = [
-            ["NlpMdsEx1.exe", "400", "100", "0", "-selfcheck"],
-            ["NlpMdsEx1.exe", "400", "100", "1", "-selfcheck"],
-            ["NlpMdsEx1.exe", "400", "100", "0", "-empty_sp_row", "-selfcheck"],
-        ]
 
-        if "+raja" in self.spec:
-            tests.extend(
-                [
-                    ["NlpMdsEx1Raja.exe", "400", "100", "0", "-selfcheck"],
-                    ["NlpMdsEx1Raja.exe", "400", "100", "1", "-selfcheck"],
-                    ["NlpMdsEx1Raja.exe", "400", "100", "0", "-empty_sp_row", "-selfcheck"],
-                ]
+    def test_N1pMDsEx1(self):
+        """Test N1pMDsEx1"""
+
+        if not self.spec.satisfies("@0.6.1") or not os.path.isdir(self.prefix.bin):
```

Comment:
@cameronrutherford mentioned this change was *introduced* in the `0.6.1` release so a version range check is in order here.

Reply:
Also, the presence of the *executable* needs to be checked in order to run it, not that the installation directory was created.